Automattic/vip-cli
# Streamline import progress output

## Description

Changes from #639 +
Flipped progress logs to use the sprite/progress logic in `/lib/cli/format.js` added in #614

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `node ./dist/bin/vip-import-sql @103 /Users/joannelee/Downloads/test.sql --search-replace="hello.com,goodbye.com"`
